stock_picking_comment_template_ext/models/stock_move.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _get_new_picking_values(self):
+        vals = super(StockMove, self)._get_new_picking_values()
+        vals["note2"] = self.sale_line_id.order_id.note2
```

Comment:
@tack-yabeQuartile Can you check if this doesn't hinder the purchase order confirmation where sale_line_id of the move is not updated.